src/content/docs/import/csv.md

```diff
@@ -54,6 +55,42 @@ The following statement will load `user.csv` into User table.
 COPY User FROM "user.csv" (header=true);
 ```
 
+## Ignoring Errors
+
+Create a node table `Person` as follows:
+
+```cypher
+create node table person (ID INT16, age INT32, PRIMARY KEY (ID));
+```
+
+The CSV file `vPerson.csv` contains the following fields (note that `2147483650` does not fit into an INT32):
+```csv
+0,4
+2,2147483650
+```
+
+The following statement will load only the first row of `vPerson.csv` into `Person` table, skipping the malformed second row.
+
+```cypher
+COPY Person FROM "vPerson.csv" (header=false, ignore_errors=true);
+```
+
+The warnings table stores any errors that were skipped during the copy.
```

Comment:
Is this really a table, tho? We're showing below that it's retrieved via a `CALL` clause, so in theory it's treated as a query result, right? We haven't described anywhere above that there's something called a warnings table, so it's important that we use the right terminology.
cc @ray6080

Reply:
Updated, let me know if the new description is good